collect_app/src/test/java/org/odk/collect/android/external/FormUriActivityTest.kt

```diff
@@ -762,7 +762,6 @@ class FormUriActivityTest {
         assertStartSavedFormIntent(null, instance.dbId, true)
     }
 
-    // TODO: Replace the explicit FormUriActivity intent with an implicit one Intent.ACTION_EDIT once it's possible https://github.com/android/android-test/issues/496
```

Comment:
Why didn't you convert it to use an implicit intent too?

Reply:
Ah sorry I meant to leave a comment and totally forgot: implicit action Intents don't work in Robolectric, but that feels ok here as we're testing an Activity rather than an action. The instrumentation tests (like `FormEditActionTest`) check that the app responds to the action.